modules.d/62bluetooth/module-setup.sh

```diff
@@ -69,7 +70,7 @@ install() {
 
         inst_multiple \
             /etc/bluetooth/main.conf \
-            /etc/dbus-1/system.d/bluetooth.conf \
+            "$dbusconfdir"/bluetooth.conf \
```

Comment:
`$dbusconfdir` is usually expanded to `/etc/dbus-1`. Shouldn't  [`$dbussystemconfdir`](https://github.com/dracutdevs/dracut/blob/master/dracut.sh#L1643) be here instead?

Reply:
Spot on Fixed.